src/translations/en.json

```diff
@@ -2393,11 +2394,6 @@
         "tabs": {
           "info": {
             "title": "Info",
-            "remove": "Remove",
-            "set": "Set",
-            "default_ui": "{action} {name} as default page on this device",
```

Comment:
Just confirming these were meant to be deleted in this PR.

Reply:
Yeah are leftovers from States UI